resources/platforms.json

```diff
@@ -1689,6 +1689,15 @@
         "Platform_Version": "11.04"
       }
     },
+    "Ubuntu_11_10": {
```

Comment:
I initially ran into a useragent that was Ubuntu 11.10 for Chromium, so I added this in anticipation of dealing with that agent. The agent ended up not being legit (was missing the `Chrome/*` part), but I figured I'd keep this here anyhow. I did fish out a Chromium Ubuntu 11.10 useragent from our logs though and added it to the test cases.